### PR TITLE
Propose ForceChangeRate

### PR DESCRIPTION
**Pull request by tongbong**
Implemented following existing PressureFlowRate

### CODE REVIEW

File: UnitsNet/Scripts/UnitDefinitions/ForceChangeRate.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿{
+  "Name": "ForceChangeRate",
+  "BaseUnit": "NewtonPerSecond",
+  "XmlDoc": "Force change rate is the ratio of the force change to the time during which the change occurred (value of force changes per unit time).",
+  "Units": [
+    {
+      "SingularName": "NewtonPerSecond",
+      "PluralName": "NewtonsPerSecond",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
```
Comment (anjdreas) [inline UnitsNet/Scripts/UnitDefinitions/ForceChangeRate.json]: We could add some of the common prefixes here, but I am not familiar with this unit and what prefixes would be commonly used.

**Comment by anjdreas** [comment]
I'll merge this in now and let you add prefixes later if you choose to do so.

**Comment by anjdreas** [comment]
Nuget 3.23 out with all your PRs.

**Comment by tongbong** [comment]
Thanks ^^

**Pull request merged**